scripts/install_pre_built_halide.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash -xe
 
 if [[ -z $Halide_to_Hardware ]]; then
-  git clone --depth 1 https://github.com/StanfordAHA/Halide-to-Hardware
-  cd Halide-to-Hardware
-
-  wget -q https://github.com/StanfordAHA/Halide-to-Hardware/releases/download/cascade-cast-issue-resolved/halide_distrib.tgz
-  tar zxf halide_distrib.tgz
+  git clone --depth 1 --branch update-lakelib-name https://github.com/StanfordAHA/Halide-to-Hardware
+  ./install_and_build_halide.sh
```

Comment:
Why do you want to build from scratch? It takes a some time to build halide, which will slow down PR status update for dependent repos.

Reply:
This was just to test whether the changes I made in Halide will work with GarnetFlow. I will delete this once the changes are merged with master.

Reply:
Sorry, this was probably not the place to test it out (but was the easiest...)

Reply:
Got it. The typically flow is have halide build a release every now and then (in its own travis build) and update the pinned release here.